dallinger/recruiters.py

```diff
@@ -280,9 +281,11 @@ def qualifications(self):
 
     def open_recruitment(self, n=1):
         """Open a connection to AWS MTurk and create a HIT."""
+        logger.info("Opening MTurk recruitment.")
         if self.is_in_progress:
-            # Already started... do nothing.
-            return None
+            raise RuntimeError(
```

Comment:
I think using the MTurkRecruiterException that's used a few lines below would be good for both clarity and consistency.

Reply:
I'm in favor of a more specific exception here, but I think that one should be reserved for raising when issues arise from making calls to the MTurk Service.

Reply:
👍 `RecruitmentAlreadyOpen`?